srtcore/group.cpp

```diff
@@ -2372,21 +2372,20 @@ int CUDTGroup::recv(char* buf, int len, SRT_MSGCTRL& w_mc)
                 // by "more correct data" if found more appropriate later. But we have to
                 // copy these data anyway anywhere, even if they need to fall on the floor later.
                 int stat;
+                char lostbuf[SRT_LIVE_MAX_PLSIZE];
+                char* msgbuf = NULL;
                 if (output_size)
                 {
                     // We have already the data, so this must fall on the floor
-                    char lostbuf[SRT_LIVE_MAX_PLSIZE];
+                    msgbuf = lostbuf;
                     stat = ps->core().receiveMessage((lostbuf), SRT_LIVE_MAX_PLSIZE, (mctrl), CUDTUnited::ERH_RETURN);
                     HLOGC(grlog.Debug,
-                          log << "group/recv: @" << id << " IGNORED data with %" << mctrl.pktseq << " #" << mctrl.msgno
-                              << ": " << (stat <= 0 ? "(NOTHING)" : BufferStamp(lostbuf, stat)));
-                    if (stat > 0)
-                    {
-                        m_stats.recvDiscard.Update(stat);
-                    }
+                          log << "group/recv: @" << id << " EXTRACTED EXTRA data with %" << mctrl.pktseq
```

Comment:
I think this LOG now deserves to display also the information about which pktseq it has extracted - and whether this packet should be IGNORED or stored as AHEAD.

Reply:
Added "TO STORE" / "TO IGNORE" to the log message.